magento/magento2
# Fix #21025 - Missing "Apply To" when editing Product attributes

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#21025: Missing "Apply To" when editing Product attributes

### Manual testing scenarios (*)

1. Store -> Attributes -> Product
2. Add New/Edit attributes
3. Expand "Advanced Attribute Properties"
4. There should be `Apply To` field editable for non-system attributes

### Questions or comments


### Contribution checklist (*)
